opencodeblocks/scene/clipboard.py

```diff
@@ -80,6 +79,8 @@ def _find_bbox_center(self, blocks_data):
         return (xmin + xmax) / 2, (ymin + ymax) / 2
 
     def _deserializeData(self, data:OrderedDict, set_selected=True):
+        if data is None: return
```

Comment:
Why this ?

Reply:
Avoid crashes then the clipboard does not contain valid JSON data.

Reply:
Yeah ok whatever